README.md

```diff
@@ -22,20 +22,25 @@ https://cloud.google.com/monitoring/agent/authorization#before_you_begin
 Role Variables
 --------------
 
-A specific version of the agent can be installed by modifying the `version`
-variable. The allowed values are `latest` (default), `MAJOR_VERSION.*.*`
-and `MAJOR_VERSION.MINOR_VERSION.PATCH_VERSION`.
+The `agent_type` is a required variable used to specify which agent is being
+configured. The available options are `monitoring`, `logging` and `ops_agent`.
 
-The agents can be configured by supplying a path to a custom configuration
-file using the variable `main_config_file`. This custom configuration file will
-overwrite the configuration file on the target VM.
+The `package_state` variable can be used to specify the desired state of the
+agent. The allowed values are `present` (default) and `absent`.
+
+The `version` variable can be used to specify which version of the agent to
```

Comment:
Some additional information that may be worth calling out:

version=latest
While this version=latest setting makes it easier to keep the agent version up to date, this setting does come with a potential risk. When a new major version is released, the policy may install the latest version of the agent from that new major release, which may introduce breaking changes. For production environments, consider using the version=MAJOR_VERSION.*.* setting below for safer agent deployments.

version=MAJOR_VERSION.*.*
When a new major release is out, this setting ensures that only the latest version from the specified major version is installed, which avoids accidentally introducing breaking changes. This is recommended for production environments to ensure safer agent deployments.

version=MAJOR_VERSION.MINOR_VERSION.PATCH_VERSION
This setting is not recommended since it prevents upgrades of new versions of the agent that include bug fixes and other improvements.

Reply:
Done.